entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md

```diff
@@ -19,6 +19,414 @@ EF7 is also available as [daily builds](https://github.com/dotnet/efcore/blob/ma
+                    ownedOwnedNavigationBuilder => ownedOwnedNavigationBuilder.OwnsMany(update => update.Commits));
+            });
+-->
+[!code-csharp[PostMetadataConfig](../../../../samples/core/Miscellaneous/NewInEFCore7/BlogsContext.cs?name=PostMetadataConfig)]
+
+With this mapping, EF7 can create and query into a complex JSON document like this:
+
+```json
+{
+  "Views": 5085,
+  "TopGeographies": [
+    {
+      "Browsers": "Firefox, Netscape",
+      "Count": 924,
+      "Latitude": 110.793,
+      "Longitude": 39.2431
+    },
+    {
+      "Browsers": "Firefox, Netscape",
+      "Count": 885,
+      "Latitude": 133.793,
+      "Longitude": 45.2431
+    }
+  ],
+  "TopSearches": [
+    {
+      "Count": 9359,
+      "Term": "Search #1"
+    }
+  ],
+  "Updates": [
+    {
+      "PostedFrom": "127.0.0.1",
+      "UpdatedBy": "Admin",
+      "UpdatedOn": "1996-02-17T19:24:29.5429092Z",
+      "Commits": []
+    },
+    {
+      "PostedFrom": "127.0.0.1",
+      "UpdatedBy": "Admin",
+      "UpdatedOn": "2019-11-24T19:24:29.5429093Z",
+      "Commits": [
+        {
+          "Comment": "Commit #1",
+          "CommittedOn": "2022-08-21T00:00:00+01:00"
+        }
+      ]
+    },
+    {
+      "PostedFrom": "127.0.0.1",
+      "UpdatedBy": "Admin",
+      "UpdatedOn": "1997-05-28T19:24:29.5429097Z",
+      "Commits": [
+        {
+          "Comment": "Commit #1",
+          "CommittedOn": "2022-08-21T00:00:00+01:00"
+        },
+        {
+          "Comment": "Commit #2",
+          "CommittedOn": "2022-08-21T00:00:00+01:00"
+        }
+      ]
+    }
+  ]
+}
+```
+
+> [!NOTE]
+> Mapping spatial types directly to JSON is not yet supported. The document above uses `double` values as a workaround. Vote for [Support spatial types in JSON columns](https://github.com/dotnet/efcore/issues/28811) if this is something you are interested in.
+
+> [!NOTE]
+> Mapping collections of primitive types to JSON is not yet supported. The document above uses a value converter to transform the collection into a comma-separated string. Vote for [Json: add support for collection of primitive types](https://github.com/dotnet/efcore/issues/28688) if this is something you are interested in.
+
+### Queries into JSON columns
+
+Queries into JSON columns work just the same as querying into any other aggregate type in EF Core. That is, just use LINQ! Here are some examples.
+
+A query for all authors that live in Chigley:
+
+<!--
+        var authorsInChigley = await context.Authors
+            .Where(author => author.Contact.Address.City == "Chigley")
+            .ToListAsync();
+-->
+[!code-csharp[AuthorsInChigley](../../../../samples/core/Miscellaneous/NewInEFCore7/JsonColumnsSample.cs?name=AuthorsInChigley)]
+
+This query generates the following SQL when using SQL Server:
+
+```sql
+SELECT [a].[Id], [a].[Name], JSON_QUERY([a].[Contact],'$')
+FROM [Authors] AS [a]
+WHERE CAST(JSON_VALUE([a].[Contact],'$.Address.City') AS nvarchar(max)) = N'Chigley'
```

Comment:
Hmm... Are we sure that we need to systematically add the CAST to nvarchar(max) everywhere where we do JSON_VALUE? At least in this case (and below) that doesn't seem necessary. In other words, 

/cc @maumar